charts/spire/templates/oidc-dp-configmap.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "spire.fullname" . }}-oidc-discovery-provider
+  namespace: {{ .Release.Namespace }}
+data:
+  oidc-discovery-provider.conf: |
+    log_level = "{{ .Values.oidc.logLevel }}"
+
+    domain = "{{ .Values.oidc.domain }}"
+    acme {
```

Comment:
- We need to figure out how to run TLS on a localhost deployment.
- Would be great if acme could be handled by other components in our infrastructure.
- How to configure the acme challenge to be used e.g. dns challenge